vulcan/hebe/model/_student.py

```diff
@@ -37,19 +37,29 @@ class Student(Serializable):
 
     @property
     def full_name(self) -> str:
-        """Gets the student's full name in "FirstName SecondName LastName" format.
+        """Gets the student's full name in "FirstName SecondName LastName" format or  "FirstName LastName" format if there is no second name.
 
         :rtype: str
         """
-        return " ".join(
-            part
-            for part in [
-                self.pupil.first_name.strip(),
-                self.pupil.second_name.strip(),
-                self.pupil.last_name.strip(),
-            ]
-            if part
-        )
```

Comment:
Keep this code, while embedding the if into the part array (like `second_name.strip() if second_name else None`).

Reply:
I can do this, but then if the student does not have a seccond name there will be a double space between the first and last names.

Reply:
No, there won't be a double space, mind the `if part` statement.